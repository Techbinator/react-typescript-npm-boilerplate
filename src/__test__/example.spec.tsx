import * as React from 'react';
import Hello from '..';
import * as TestRenderer from 'react-test-renderer';

describe('Component', () => {
  it("should show text 'Hello World' with className demo", () => {
    const component = TestRenderer.create(<Hello text="World" />);
    const testInstance = component.root;

    expect(testInstance.findByType(Hello).props.text).toBe('World');

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
