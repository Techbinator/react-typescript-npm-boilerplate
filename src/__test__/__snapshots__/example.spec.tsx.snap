// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should show text 'Hello World' with className demo 1`] = `
<div
  className="demo"
>
  Hello 
  World
</div>
`;
